capstone/cite/views.py

```diff
@@ -81,7 +81,7 @@ def series(request, series_slug):
         .order_by('full_name'))
     if not reporters:
         raise Http404
-    reporters = [(reporter, sorted(reporter.volumes.all(), key=lambda volume: natural_sort_key(volume.volume_number))) for reporter in reporters]
+    reporters = [(reporter, sorted(reporter.volumes.all().exclude(out_of_scope=True), key=lambda volume: natural_sort_key(volume.volume_number))) for reporter in reporters]
```

Comment:
You want to put this .exclude() up in line 80 so it doesn't result in a separate sql query. The `prefetch_related` is making sure that these `reporter.volumes.all()` calls won't actually result in a new database hit, but only if they're unfiltered -- hence the change in `django_assert_num_queries`. (Thanks, tests!)

Looking at this again, prefetch_related is totally a premature optimization -- there will only ever be one or two reporters anyway, so using prefetch_related only reduces our query count by like one and I shouldn't have bothered. It would also be valid to move all the filters down to line 84 and get rid of the prefetch_related for the sake of simplicity, and bump django_assert_num_queries to 3. 🤷‍♂️

Reply:
Ah, OK, thanks. I just pushed another test "correction", but will go back and do it this way.